src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        ColorSupplier coloursSupplier = new ColorSupplier();
+        Colors randomColor = coloursSupplier.getRandomColor();
+        randomBall.setColor(randomColor.toString());
```

Comment:
```suggestion
        randomBall.setColor(randomColor.name());
```
enums have this method to get name, better use this instead